prow/tide/tide.go

```diff
@@ -1183,6 +1195,37 @@ func (c *Controller) mergePRs(sp subpool, prs []PullRequest) error {
 	return fmt.Errorf("failed merging %v%s: %v", failed, batch, utilerrors.NewAggregate(errs))
 }
 
+// ensureTideContext makes sure we set the tide github context to PASSING for each PR we're about
+// to merge
+func (c *Controller) ensureTideContext(pr PullRequest, log *logrus.Entry) error {
+	var actualState githubql.StatusState
+	contexts, err := headContexts(log, c.ghc, &pr)
+	if err != nil {
+		return err
+	}
+	for _, ctx := range contexts {
+		if string(ctx.Context) == "tide" {
+			actualState = ctx.State
+		}
+	}
+
+	if string(actualState) != "success" {
+		if err := c.ghc.CreateStatus(
```

Comment:
I think it would preferable to just do this unconditionally, because:
* Uses always one token vs uses always one and up to two tokens
* Quicker, because we save an api call
* Simpler code

Reply:
makes sense, will remove the check for current state